NOTICE

```diff
@@ -0,0 +1,12 @@
+
+Devicetree Specification
```

Comment:
Same as this part, is this also planned as the update for the next patch?

Reply:
fixed. I need to sort out the license though